frontend/src/components/BaseInput.vue

```diff
@@ -120,12 +90,6 @@ export default defineComponent({
       default: undefined,
     },
 
-    counter: {
```

Comment:
This `counter` prop was used way back when we only generated a 16 byte random number and allowed a 16 byte memo. The counter was used to show how much more text you can fit in the memo. Since that was removed because it reduced security a lot, we no longer use this `counter` prop anywhere, so I deleted it